week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>First do an example before starting the questions</p><p>I recommend a problem similar to the questions but keep it clean and simple. Such as: bird is flying NE at 6 m/s, breakdown and show the x,y components using the Pythagorean theorem. Then the bird hits a head-wind of 2 m/s; what is the bird's new velocity (in x,y, and compass coord)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>